platform-sdk/swirlds-common/src/main/java/com/swirlds/common/merkle/synchronization/stats/ReconnectMapStats.java

```diff
@@ -27,6 +27,11 @@
  * is ready, or in tests.
  */
 public interface ReconnectMapStats {
+    /**
+     * A no-operations implementation of the ReconnectMapStats interface that does nothing.
+     */
+    ReconnectMapStats NO_OP = new ReconnectMapStats() {};
```

Comment:
IMO we shouldn't have this. Unit tests can handle making a mock metric object

Reply:
See above. But okay. Removed.